bot/handlers/admin.py

```diff
@@ -218,6 +236,133 @@ async def delete_admin(
         await state.set_state(FSMSuperAdmin.admin_management_menu)
 
 
+@router.message(lambda m: m.text == _("📽️Add video"), StateFilter(FSMSuperAdmin.main_menu, FSMAdmin.main_menu))
+async def process_add_video_button(message: Message, state: FSMContext):
+    await message.delete()
+    await message.answer(text=_("Please enter the name of the cover"))
+    await state.set_state(FSMAdmin.fill_cover_name)
+
+
+@router.message(StateFilter(FSMAdmin.fill_cover_name))
+async def process_cover_name_input(message: Message, state: FSMContext):
+    await message.answer(text=_("Please enter the link to the cover"))
+    await state.update_data(cover_name=message.text)
+    await state.set_state(FSMAdmin.fil_cover_url)
+
+
+@router.message(StateFilter(FSMAdmin.fil_cover_url))
+async def process_cover_url_input(message: Message, state: FSMContext):
+    await message.answer(text=_("Select gender"), reply_markup=ChooseGenderKeyboard()())
+    await state.update_data(cover_url=message.text)
+    await state.set_state(FSMAdmin.fill_cover_gender)
+
+
+@router.message(StateFilter(FSMAdmin.fill_cover_gender))
+async def process_cover_gender_input(message: Message, state: FSMContext):
+    await message.answer(text=_("Enter the number of participants"))
+    if message.text == _("Female"):
+        await state.update_data(cover_gender=False)
+    else:
+        await state.update_data(cover_gender=True)
+    await state.set_state(FSMAdmin.fill_cover_members)
+
+
+@router.message(StateFilter(FSMAdmin.fill_cover_members))
+async def process_cover_members_input(message: Message, state: FSMContext):
+    await message.answer(text=_("Choose difficulty level"), reply_markup=ChooseDifficultKeyboard()())
+    await state.update_data(cover_members=message.text)
```

Comment:
members field is "3, 4, ..., 9, >9" number
Check it

Reply:
🔥